### PR TITLE
new variable for button hover for  design=`tab`

### PR DESCRIPTION
**Pull request by vesper2000**
# patch

### CODE REVIEW

File: scss/frost-theme.scss [hunk 1]
```diff
@@ -37,6 +37,9 @@ $frost-color-button-tertiary-bg: inherit !global;
 $frost-color-button-tertiary-hover-bg: rgba($frost-color-blue-1, 0.1) !global;
 $frost-color-button-tertiary-active: $frost-color-blue-3 !global;
 
+// Buttons - design
+$frost-color-button-tab-hover-bg: rgba($frost-color-blue-1, 0.05) !global;
```
Comment (sglanzer-fire) [inline scss/frost-theme.scss]: Let's make sure that we need a different hover (0.05) vs. the tertiary (0.1) - ideally we could use a common variable

**Comment by vesper2000** [comment]
Phil confirmed it should be the same, but the existing hover value needs to change to .05.

**Comment by sglanzer-fire** [comment]
Excellent, thanks for confirming :)

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-theme/pull-request/41/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-theme)

**Pull request merged**